packages/thirdweb/src/extensions/prebuilts/deploy-vote.ts

```diff
@@ -0,0 +1,186 @@
+import type { Chain } from "src/chains/types.js";
+import type { ThirdwebClient } from "../../client/client.js";
+import { type ThirdwebContract, getContract } from "../../contract/contract.js";
+import { deployViaAutoFactory } from "../../contract/deployment/deploy-via-autofactory.js";
+import { getOrDeployInfraForPublishedContract } from "../../contract/deployment/utils/bootstrap.js";
+import type { FileOrBufferOrString } from "../../storage/upload/types.js";
+import type { Prettify } from "../../utils/type-utils.js";
+import type { ClientAndChainAndAccount } from "../../utils/types.js";
+import { decimals } from "../erc20/read/decimals.js";
+
+/**
+ * @extension PREBUILT
+ */
+export type VoteContractParams = {
+  name: string;
+  /**
+   * The contract address for the ERC20 that will be used as voting power
+   */
+  token: string;
+  /**
+   * The number of blocks after a proposal is created that voting on the proposal starts.
+   * A block is a series of blockchain transactions and occurs every ~1 seconds.
+   * Block time is different across EVM networks
+   *
+   * Defaults to 0 (zero)
+   */
+  initialVotingDelay?: bigint;
+  /**
+   * The number of blocks that voters have to vote on any new proposal.
+   */
+  initialVotingPeriod: bigint;
+  /**
+   * The minimum number of voting tokens a wallet needs in order to create proposals.
+   * This amount that you have to enter is _not_ in wei. If you want users to have a least 0.5 ERC20 token to create proposals,
+   * enter `"0.5"`. The deploy script will fetch the ERC20 token's decimals and do the unit conversion for you.
+   */
+  initialProposalThreshold: string;
+  /**
+   * The fraction of the total voting power that is required for a proposal to pass.
+   * A value of 0 indicates that no voting power is sufficient,
+   * whereas a value of 100 indicates that the entirety of voting power must vote for a proposal to pass.
+   */
+  initialVoteQuorumFraction: bigint;
```

Comment:
again not a bigint if its only from 0 to 100. the variable name is very vague too, maybe `minVoteQuorumRequiredPercent` ?